src/scss/settings/checkout/customer/_settings.scss

```diff
@@ -5,4 +5,3 @@
 $customerEmail-action-margin: spacing("single") + spacing("fifth") 0 0 0;
 $customerEmail-button-padding: spacing("half") + spacing("sixth") spacing("third");
 $customerView-actions-marginTop: remCalc(-5px);
-$customerView-actions-marginTop--large: remCalc(-40px);
```

Comment:
Can we move `0` here so that we can have one single centralised setting file to config the view?

Reply:
Also, it looks like the Jira ticket number is missing in the PR title. Can you please always add it to the PR title? 🙇🏻‍♂️